faq.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
```

Comment:
<script src="https://code.jquery.com/jquery-3.4.0.js"></script>

This is the script tag that should be used for jquery. Some reason the slim.min does not support ajax calls. Also, please move all script tags on all pages to the bottom of the page, just about the </body>